### PR TITLE
Document available airnode examples

### PR DESCRIPTION
**Pull request by dcroote**
[AN-387](https://api3dao.atlassian.net/browse/AN-387)

### CODE REVIEW

File: packages/airnode-examples/README.md [hunk 1]
```diff
@@ -16,12 +16,26 @@ but you may adapt the configuration to work for your target chain.
...
 
-If you would like to know more about RRP, read the [API3 docs](https://docs.api3.org/airnode/v0.2/concepts/).
+If you would like to know more about RRP, read the [API3 docs](https://docs.api3.org/airnode/v0.3/concepts/).
+
+## Available examples
+
+We call examples "integrations" because they are integrated with some API. With each integration you are offered a
+choice of how you want to run the Airnode and which network to use, as mentioned above. The integrations have been
+designed to highlight various Airnode functionality and use cases, from simple price requests to more complex
+authenticated requests encoding multiple reserved parameters. The following list orders integrations by increasing
```
Comment (Siegrift) [inline packages/airnode-examples/README.md:32]: Order by complexity is a subjective thing and will be hard to follow as the list grows in size. I think we can just use the folder names in aplhabetical order.

Comment (Siegrift) [reply packages/airnode-examples/README.md:32]: These folder names would ideally be links.

Comment (Siegrift) [reply packages/airnode-examples/README.md:32]: And ideally there would be a unit test that would verify that all of them are listed.

File: packages/airnode-examples/README.md [hunk 2]
```diff
@@ -36,8 +50,7 @@ run the example with Rinkeby.
 
 ### 1. Choose an example
 
-First, you need to choose what example you want to run. We call these examples "integrations" because they are
```
Comment (Siegrift) [inline packages/airnode-examples/README.md:39]: 👍

**Review by Siegrift — CHANGES_REQUESTED** [review]
Be sure to mention that you need to include the new integration in the root examples readme in https://github.com/api3dao/airnode/tree/master/packages/airnode-examples/integrations#adding-new-integration

File: packages/airnode-examples/README.md (hunk 1 above)
Comment (Siegrift) [reply packages/airnode-examples/README.md:32]: > And ideally there would be a unit test that would verify that all of them are listed.

This is probably an overkill since this will be migrated in the docs.

Comment (dcroote) [reply packages/airnode-examples/README.md]: Thanks for the feedback. I made the requested updates:
- Changed to alphabetical
- Added links
- Mentioned integration addition instructions

**Review by Siegrift — APPROVED** [review]
LGTM
